src/components/productRow/index.js

```diff
@@ -0,0 +1,14 @@
+import RowItem from "../rowItem";
+import "./productRow.css";
+
+function ProductRow(props) {
```

Comment:
Don't handle presentation using JS, you can handle it using CSS. Handling it like this means you will have 5 products on each row even on a smaller screen of a mobile. You should add all products in one container and then handle the number of products per row using CSS based on screen width.